lib/validator/validator.go

```diff
@@ -256,6 +312,35 @@ func validateProtocol(v *validator.Validate, structLevel *validator.StructLevel)
 	}
 }
 
+func validateIPv4Address(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
```

Comment:
Ah I see... maybe these should be validate IPOrCIDRv4 or something similar.

Reply:
yeah good idea, will rename our validator functions.